tests/acceptance/course-page/code-examples/export-to-github-test.js

```diff
@@ -0,0 +1,173 @@
+import catalogPage from 'codecrafters-frontend/tests/pages/catalog-page';
+import codeExamplesPage from 'codecrafters-frontend/tests/pages/course/code-examples-page';
+import coursePage from 'codecrafters-frontend/tests/pages/course-page';
+import createCommunityCourseStageSolution from 'codecrafters-frontend/mirage/utils/create-community-course-stage-solution';
+import percySnapshot from '@percy/ember';
+import testScenario from 'codecrafters-frontend/mirage/scenarios/test';
+import { setupAnimationTest } from 'ember-animated/test-support';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
+import { signIn } from 'codecrafters-frontend/tests/support/authentication-helpers';
+import { settled } from '@ember/test-helpers';
+import courseOverviewPage from 'codecrafters-frontend/tests/pages/course-overview-page';
+
+module('Acceptance | course-page | code-examples | export-to-github', function (hooks) {
+  setupApplicationTest(hooks);
+  setupAnimationTest(hooks);
+
+  async function setupTestScenario(server) {
```

Comment:
The pattern to do this in q-unit is to do `hooks.beforeEach` (search the codebase to find usages)